### PR TITLE
move more basic utils to plone.base

### PR DESCRIPTION
**Pull request by jensens**
needed i.e. to untangle plone.app.content

### CODE REVIEW

File: Products/CMFPlone/PloneTool.py [hunk 1]
```diff
@@ -913,7 +915,7 @@ def listMetaTags(self, context):
         return result
 
     @security.public
-    def getUserFriendlyTypes(self, typesList=None):
+    def getUserFriendlyTypes(self, typesList=[]):
```
Comment (mauritsvanrees) [inline Products/CMFPlone/PloneTool.py:918]: No no no, don't do this. Classic error:

```
>>> def foo(bar=[]):
...     bar.append(1)
...     print(bar)
... 
>>> foo()
[1]
>>> foo()
[1, 1]
>>> foo()
[1, 1, 1]
>>> foo()
[1, 1, 1, 1]
```

Same remark is true for `plone.base` where you copied this to.

Comment (jensens) [reply Products/CMFPlone/PloneTool.py:918]: Sorry, in fact I know this.... but brain-malfunction.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
this was superseeded by other PRs

**Pull request closed without merging**